privacy.adoc

```diff
@@ -351,7 +351,7 @@ can only increase with layers on top, but that doesn't seem to be the
 case for privacy, which can be improved or made worse arbitrarily in
 layers on top. Why is that? Any layer on top of Bitcoin, as explained
 in <<_layered_scaling,Layered scaling>>, must use on-chain
```

Comment:
<<_layered_scaling>>  (If we want to go for "technical" xrefs rather than descriptive).